dev/releases/make_archives.py

```diff
@@ -210,6 +227,7 @@
     with open(manifest_filename, 'w') as manifest:
         for archive in archives_to_create:
             manifest.write(f"{archive}\n")
+        manifest.write("package-infos.json\n")
```

Comment:
The filename here doesn't seem to match the one used above?

Reply:
No sorry, it does, I got confused

Reply:
I think that's correct. The `PackageInfos-to-JSON.g` generates the file `package-infos.json`.